lib/tessel/deployment/rust.js

```diff
@@ -159,6 +165,9 @@ exportables.remoteRustCompilation = (opts) => {
         });
     });
 
+    // Start the spinner to folks know things are happening
+    log.spinner.start();
```

Comment:
The `spinner` is started in `bin/tessel-2.js` -> https://github.com/tessel/t2-cli/blob/master/bin/tessel-2.js#L87